wpilibNewCommands/src/main/native/include/frc2/command/Command.h

```diff
@@ -318,6 +318,18 @@ class Command : public wpi::Sendable, public wpi::SendableHelper<Command> {
   [[nodiscard]]
   CommandPtr FinallyDo(std::function<void(bool)> end) &&;
 
+  /**
+   * Decorates this command with a lambda to call on interrupt or end, following
+   * the command's inherent Command::End(bool) method. The provided lambda will
+   * run identically in both interrupt and end cases.
+   *
+   * @param end a lambda to run when the command ends, whether or not it was
+   * interrupted.
+   * @return the decorated command
+   */
+  [[nodiscard]]
+  CommandPtr FinallyDo(std::function<void(void)> end) &&;
```

Comment:
for consistency, this should probably just be `void()`. Looks like `HandleInterrupted` is an outlier here.